rbd-target-api.py

```diff
@@ -177,6 +178,25 @@ def get_sys_info(query_type=None):
         # Request Unknown
         return jsonify(message="Unknown /sysinfo query"), 404
 
+def _parse_target_controls(controls_json):
+    controls = {}
+    raw_controls = json.loads(controls_json)
+    for k in settings.Settings.GATEWAY_SETTINGS:
+        if raw_controls.has_key(k):
```

Comment:
has_key has been deprecated in favor of 'in' (I think).
See http://portingguide.readthedocs.io/en/latest/dicts.html

Reply:
This is already merged, so feel free to open a PR to incorporate your suggestions.